reference/catalog/product-modifiers_catalog.v3.yml

```
@@ -394,7 +460,7 @@ paths:
                     default_value:
                       type: string
                       description: |
-                        (date, text, multi_line_text, numbers_only_text) The default value. Shown on a date option as an ISO-8601–formatted string, or on a text option as a string.
+                        (date, text, multi_line_text, numbers_only_text) The default value. Shown on a date option as an ISO-8601–formatted string, or on a text option as a string. See [Config](#config) for more details.
```

Looks like you named the subheading `Configs`. If so, this should be [Configs](#configs)
```suggestion
                        (date, text, multi_line_text, numbers_only_text) The default value. Shown on a date option as an ISO-8601–formatted string, or on a text option as a string. See [Configs](#configs) for more details.
```

---

Alternatively you can switch the heading to `Config`, which would let you resolve all these comments in one swell foop